src/Action/LtiInstance/CreateAction.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ *  This program is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License
+ *  as published by the Free Software Foundation; under version 2
+ *  of the License (non-upgradable).
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ *  Copyright (c) 2022 (original work) Open Assessment Technologies S.A.
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\Action\LtiInstance;
+
+use OAT\SimpleRoster\Entity\LtiInstance;
+use OAT\SimpleRoster\Events\LtiInstanceUpdated;
+use OAT\SimpleRoster\Repository\Criteria\LtiInstanceCriteria;
+use OAT\SimpleRoster\Repository\LtiInstanceRepository;
+use OAT\SimpleRoster\Request\Validator\LtiInstance\Validator;
+use OAT\SimpleRoster\Responder\LtiInstance\Serializer;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface as EventDispatcher;
+
+class CreateAction
```

Comment:
It would be better if the name of the class fully reflects its purpose. What if we add CRUD for some other instances? I'd suggest to name the class like `class LtiInstanceCreateAction`. The same applies to other actions.  `LtiInstanceListAction`, `LtiInstanceUpdateAction`, `LtiInstanceDeleteAction`

Reply:
fixed